Stratis.Bitcoin/Stratis.Bitcoin.csproj

```diff
@@ -18,7 +18,11 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Ah new to git ;)
You are suppose to select whats inside the git signals
either whats inside HEAD or whats in Up Stream

This
```
+<<<<<<< HEAD
 +    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
 +=======
      <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.0.0" />
 +>>>>>>> upstream/master
```
Becomes
```
<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
```